ETWAnalyzer/EventDump/DumpMemory.cs

```diff
@@ -362,23 +380,40 @@ private void Print(List<Match> matches)
-                    ColorConsole.WriteEmbeddedColorLine($"[yellow]{m.Process.GetProcessWithId(UsePrettyProcessName)}[/yellow][grey]{GetProcessTags(m.Process, m.SessionStart)}[/grey] {(NoCmdLine ? "" : m.CmdLine)} ", ConsoleColor.DarkCyan, true);
-                    ColorConsole.WriteEmbeddedColorLine($"[red]{moduleInfo}[/red]");
+                    if (!IsFileTotalMode)
+                    {
+                        ColorConsole.WriteEmbeddedColorLine($"[darkcyan]{GetDateTimeString(m.SessionEnd, m.SessionStart, TimeFormatOption)}[/darkcyan] " +
+                            $"[{GetColor(m.DiffMb)}]Diff: {m.DiffMb,4}[/{GetColor(m.DiffMb)}] " +
+                            $"[{GetColorTotal(m.CommitedMiB)}]Commit {m.CommitedMiB,4} MiB[/{GetColorTotal(m.CommitedMiB)}] " +
+                            $"[{GetColorTotal(m.WorkingSetMiB)}]WorkingSet {m.WorkingSetMiB,4} MiB[/{GetColorTotal(m.WorkingSetMiB)}] " +
+                            (ShowDetails ? $"[{GetColorTotal(m.WorkingsetPrivateMiB)}]WorkingsetPrivate {m.WorkingsetPrivateMiB,4} MiB[/{GetColorTotal(m.WorkingsetPrivateMiB)}] " : "") +
+                            $"[{GetColorTotal(m.SharedCommitInMiB)}]Shared Commit: {m.SharedCommitInMiB,4} MiB [/{GetColorTotal(m.SharedCommitInMiB)}] ", null, true);
+                        ColorConsole.WriteEmbeddedColorLine($"[yellow]{m.Process.GetProcessWithId(UsePrettyProcessName)}[/yellow][grey]{GetProcessTags(m.Process, m.SessionStart)}[/grey] {(NoCmdLine ? "" : m.CmdLine)} ", ConsoleColor.DarkCyan, true);
+                        ColorConsole.WriteEmbeddedColorLine($"[red]{moduleInfo}[/red]");
+                    }
+                    
                     printedFiles++;
                 }
 
                 if (IsSummary && printedFiles > 1)
                 {
-                    ColorConsole.WriteEmbeddedColorLine($"[cyan]Memory Total per File:[/cyan] [{GetTrendColor(totalDiff)}]TotalDiff: {totalDiff} [/{GetTrendColor(totalDiff)}] [{GetColorTotal(totalCommitedMemMiB)}] TotalCommitedMem: {totalCommitedMemMiB} MiB [/{GetColorTotal(totalCommitedMemMiB)}] [Darkyellow] Number of Involved Processes: {processCount} [/Darkyellow]");
+                    ColorConsole.WriteEmbeddedColorLine($"[cyan]Memory Total per File:[/cyan] [{GetTrendColor(totalDiff)}]" +
+                        $"TotalDiff: {totalDiff} [/{GetTrendColor(totalDiff)}] " +
+                        $"[{GetColorTotal(totalCommitedMemMiB)}] TotalCommited: {totalCommitedMemMiB} MiB [/{GetColorTotal(totalCommitedMemMiB)}] " +
+                        (IsFileTotalMode ? 
+                            $"[{GetColorTotal(totalWorkingsetPrivateMemMiB)}] TotalWorkingsetPrivate: {totalWorkingsetPrivateMemMiB} MiB [/{GetColorTotal(totalWorkingsetPrivateMemMiB)}]" : 
```

Comment:
The totals should be width aligned. Like {totalDiff}  -> {totalDiff,6} , {totalCommitedMemMiB} -> {totalCommitedMemMiB,6} {totalWorkingsetPrivateMemMiB}  -> {totalWorkingsetPrivateMemMiB,6}